eng/pipelines/runtimelab/install-wasi-sdk.ps1

```diff
@@ -1,8 +1,27 @@
-Invoke-WebRequest -Uri https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-22/wasi-sdk-22.0.m-mingw64.tar.gz -OutFile wasi-sdk-22.0.m-mingw64.tar.gz
+[CmdletBinding(PositionalBinding=$false)]
+param(
+    $InstallDir,
+    [switch]$CI
+)
 
-tar -xzf wasi-sdk-22.0.m-mingw64.tar.gz
+Set-Location -Path $InstallDir
 
-mv wasi-sdk-22.0+m wasi-sdk
+if ($IsWindows)
+{
+    $WasiTar = "wasi-sdk-22.0.m-mingw64.tar.gz"
+    $WasiFolder = "wasi-sdk-22.0+m"
+}
+else
+{
+    $WasiTar = "wasi-sdk-22.0-linux.tar.gz"
+    $WasiFolder = "wasi-sdk-22.0"
+}
+
+Invoke-WebRequest -Uri https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-22/$WasiTar -OutFile $WasiTar
```

Comment:
please add `$ProgressPreference = 'SilentlyContinue'` otherwise `Invoke-WebRequest` is very slow.
See https://stackoverflow.com/questions/28682642/powershell-why-is-using-invoke-webrequest-much-slower-than-a-browser-download/43477248#43477248

Reply:
Thanks, done.  I will squash all this when I've finished cleaning things up.